src/Jetpack_Usage.php

```diff
@@ -0,0 +1,161 @@
+<?php
+namespace Automattic\Jetpack\Tracking;
+
+use Automattic\Jetpack\Tracking;
```

Comment:
It's worth noting that having the same class name as the namespace we're in, and that contributes to the confusion that could introduce with regards to my comment above.

Reply:
it is not longer the case since https://github.com/Automattic/jetpack/pull/12687/commits/2f92d93516a41c1ac97771e22e6c03df0e0bcacf